src/lib/ext-monitor.js

```diff
@@ -10,9 +10,8 @@ export default class ExtensionMonitor {
 
   async getAllExtensions() {
     const extensions = await browser.management.getAll();
-    const self = await browser.management.getSelf();
     return extensions.filter((extension) => {
-      return extension.type === 'extension' && extension.id !== self.id;
+      return extension.type === 'extension' && extension.id !== this.selfId;
```

Comment:
Is there a reason for storing the `browser.runtime.id` constant in a new member instead of directly using `browser.runtime.id`?

Reply:
Not any strong reason, I just thought I am using the extension id two times, so, stored it in a variable and used the same variable twice. Now, I have updated it by using directly.